molgenis-core-ui/src/main/javascript/modules/react-components/AttributeControl.js

```diff
@@ -61,22 +74,22 @@
 				
 				switch(attr.fieldType) {
```

Comment:
Why is there a big switch here?
I would expect that
- Controls have all the logic they need to render themselves from a attributemetdata
- New controls can register themselves
(so then this becomes extensible by new data/control types)